src/arch/aarch64/kernel/start.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2020 Stefan Lankes, RWTH Aachen University
+//
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+
+use crate::arch::aarch64::kernel::serial::SerialPort;
+use crate::arch::aarch64::kernel::BootInfo;
+use crate::KERNEL_STACK_SIZE;
+
+static mut BOOT_STACK: [u8; KERNEL_STACK_SIZE] = [0; KERNEL_STACK_SIZE];
+
+/// Entrypoint - Initalize Stack pointer and Exception Table
+#[inline(never)]
+#[no_mangle]
+#[naked]
+pub unsafe extern "C" fn _start() -> ! {
+	asm!("ldr x1, {0}",
+		 "add x1, x1, {1}",
+		 "sub x1, x1, #0x10",	/* Previous version subtracted 0x10 from End, so I'm doing this too. Not sure why though. COMMENT from SL: This is a habit of mine. I always start 16 bytes before the end of the stack. */
```

Comment:
It might be cleaner to do this `const` argument of this `asm!` macro using the already documented [`TaskStacks::MARKER_SIZE`](https://github.com/hermitcore/libhermit-rs/blob/3f93b2f90b1d35955eae5459935b2bfe861a01c6/src/arch/x86_64/kernel/scheduler.rs#L87-L90).

See [`x86_64::_start`](https://github.com/hermitcore/libhermit-rs/blob/e30332a53132c8ad69436b84333ee4469644692f/src/arch/x86_64/kernel/start.rs)

Reply:
Ok, I introduced alse a constant TaskStacks::MARKER_SIZE in our aarch64 port.